reporter/src/main/kotlin/reporters/NoticeReporter.kt

```diff
@@ -114,29 +115,38 @@ class NoticeReporter : Reporter() {
                 licenseConfiguration
             ).run(postProcessingScript)
         } else {
-            val processedFindings = licenseFindings.removeGarbage(copyrightGarbage).processStatements()
-            NoticeReport(listOf(header), processedFindings, emptyList())
+            NoticeReport(listOf(header), licenseFindings, emptyList())
         }
 
         outputStream.bufferedWriter().use {
-            it.write(generateNotices(noticeReport, licenseTextProvider))
+            it.write(generateNotices(noticeReport, licenseTextProvider, copyrightGarbage))
         }
     }
 
-    private fun getLicenseFindings(ortResult: OrtResult): LicenseFindingsMap =
-        ortResult
-            .collectLicenseFindings(omitExcluded = true)
-            .values
-            .flatMap { licenseFindingsForPackage -> licenseFindingsForPackage.keys }
-            .groupBy({ it.license }, { it.copyrights.map { copyrightFinding -> copyrightFinding.statement } })
-            .mapValues { (_, copyrightStatements) -> copyrightStatements.flatten().toMutableSet() }
-            .toSortedMap()
+    private fun getLicenseFindings(ortResult: OrtResult): Map<Identifier, LicenseFindingsMap> =
```

Comment:
It's kind of out of scope, but just as a hint: as we're moving towards SDPX expression this mapping will stop work as soon as we have keys like `MIT AND Apache-2.0' for detect license findings / curations. So let's try to not create too many additional dependency on the presence of such mapping.

Reply:
But we will still be able to map detected copyrights to a specific license.